RUFAS/routines/animal/pen.py

```diff
@@ -517,7 +512,11 @@ def calc_daily_walking_dist(self):
         """
         Sets the daily walking distance for the cows in the pen (if any).
         """
-        if "Cow" in self.classes_in_pen:
+        if AnimalType.DRY_COW or AnimalType.LAC_COW in list(self.animal_grouping_scenario.values()):  # CHECK
+            for animal in list(self.animals_in_pen.values()):
+                if type(animal).__name__ == "Cow":
+                    animal.calc_daily_walking_dist(self.vertical_dist_to_parlor, self.horizontal_dist_to_parlor)
+        if self.animal_combination == 2 or self.animal_combination == 3 or self.animal_combination == 4:  # CHECK
             for animal in list(self.animals_in_pen.values()):
                 if type(animal).__name__ == "Cow":
                     animal.calc_daily_walking_dist(self.vertical_dist_to_parlor, self.horizontal_dist_to_parlor)
```

Comment:
```suggestion
        should_calc_walking_distance = (AnimalType.DRY_COW or AnimalType.LAC_COW in list(self.animal_grouping_scenario.values()) or self.animal_combination == 2 or self.animal_combination == 3 or self.animal_combination == 4
        if shoul_calc_walking_distance:  # CHECK
            for animal in list(self.animals_in_pen.values()):
                if type(animal).__name__ == "Cow":
                    animal.calc_daily_walking_dist(self.vertical_dist_to_parlor, self.horizontal_dist_to_parlor)
```
Why does it say "CHECK" here? Also the if statements can be combined.

Reply:
The two if statements are two potential implementations that I made (I was going to pick one) which is why they both have "check" on them.

Reply:
Ok. In situations like that it is important to discuss the options with whoever can provide insight, then decide how you're gonna implement something, then implement it, then open up the PR for review. If you still unsure about it, then you can leave a comment in the PR and ask reviewers specifically if it would be better to do xyz there, for reasons abc.